projects/lib/src/dynamic-views/item/item.component.html

```diff
@@ -5,7 +5,7 @@
 
 <ng-container
   *ngIf="!config.type.startsWith('_') && condition"
-  [ngTemplateOutlet]="zoneRef.templates[config.type || id]?.templateRef"
```

Comment:
Removing this `?` is dangerous, because you could have some configuration referencing a template that does not exist.

Reply:
same here